source/NVDAObjects/window/excel.py

```diff
@@ -817,7 +915,12 @@ def _getFormatFieldAndOffsets(self,offset,formatConfig,calculateOffsets=True):
 			except COMError:
 				pass
 			try:
-				formatField['background-color']=colors.RGB.fromCOLORREF(int(self.obj.excelCellObject.interior.color))
+				formatField['background-pattern']=backgroundPatternLabels.get(cellObj.Interior.Pattern)
+				if formatField['background-pattern']=="linear gradient" or formatField['background-pattern']=="rectangular gradient":

```

Comment:
This is going to break in languages other than English because the label will get translated for those. I suggest you instead do something like this:

```
pattern = cellObj.Interior.Pattern
formatField['background-pattern'] = backgroundPatternLabels.get(pattern)
if pattern in (xlPatternLinearGradient, xlPatternRectangularGradient):
	...# Other stuff here
```